osu.Game/Overlays/Social/Header.cs

```diff
@@ -55,8 +55,8 @@ public enum SocialTab
     {
         [Description("Online Players")]
         OnlinePlayers,
-        //[Description("Online Friends")]
-        //OnlineFriends,
+        [Description("Online Friends")]

```

Comment:
Let's remove the "Online" part from these tabs for now, since it's not actually checking online status.